File: 1051.py
# Author: Zoljargal Gantumur
# Runtime: 36 ms, faster than 60.90% of Python3 online submissions
# Memory Usage: 14 MB, less than 19.71% of Python3 online submissions

class Solution:
    def heightChecker(self, heights: List[int]) -> int:
        asd = heights.copy()
        asd.sort()
        ans = 0
        for i in range(len(asd)):
            if asd[i] != heights[i]:
                ans+=1
        return ans

File: 1051.cpp
/*
	Author: Zoljargal Gantumur
	Runtime: 4 ms, faster than 82.57% of C++ online submissions
	Memory Usage: 8.3 MB, less than 31.52% of C++ online submissions
*/

class Solution {
public:
    int heightChecker(vector<int>& heights) {
        vector<int> v = heights;
        sort(v.begin(), v.end());
        int ans=0;
        for(int i=0; i<v.size(); i++){
            if(v[i]!=heights[i])
                ans++;
        }
        return ans;
    }
};

